django/conf/global_settings.py

```diff
@@ -473,6 +474,14 @@ def gettext_noop(s):
 SESSION_FILE_PATH = None
 # class to serialize session data
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
+# Whether to hash the session_key when stored in the backend
+SESSION_STORE_KEY_HASH = True
+# Whether to allow non-hashed session keys to be loaded. Setting this to
+# 'False' bypasses the security benefit of hashing session keys. This
+# should be set to 'True' after moving session keys from unhashed to
+# hashed. The default value is 'False' for version below 4.0 and True
+# starting from version 4.0
+SESSION_REQUIRE_KEY_HASH = (VERSION >= (4, 0))
```

Comment:
If we want to change the default we can add a separate ticket and change it in Django 4.0. There is no need to use `VERSION`.